django/restapi/urls.py

```diff
@@ -7,8 +9,10 @@
 # Additionally, we include login URLs for the browsable API.
 
 urlpatterns = [
-    path('login', views.login),
-    path('register', views.register),
+    path('login', views.LoginView.as_view(), name='knox_login'),
+    path('logout', knox_views.LogoutView.as_view(), name='knox_logout'),
+    path('logoutall', knox_views.LogoutAllView.as_view(), name='knox_logoutall'),
```

Comment:
Do we really need an endpoint to log out all users? Or did I get something wrong?

Reply:
It logs out all sessions of the current user